tests/finance/test_slippage.py

```diff
@@ -734,10 +737,10 @@ def test_orders_stop_limit(self):
                 'limit': 3.4})
         ]
 
-        bar_data = BarData(self.data_portal,
-                           lambda: self.minutes[0],
-                           self.sim_params.data_frequency,
-                           self.trading_calendar)
+        bar_data = self.create_bardata(
+            simulation_dt_func=lambda: self.minutes[0],
+            data_frequency=self.sim_params.data_frequency
```

Comment:
It's true that only `self.sim_params.data_frequency` is ever passed in this test suite. But what about the other suites that use this fixture and want to use a frequency other than `self.sim_params.data_frequency`?